src/shared/tpm2-util.c

```diff
@@ -3507,36 +3507,42 @@ static int tpm2_build_sealing_policy(
 }
 
 #if HAVE_OPENSSL
-static int tpm2_ecc_curve_from_openssl_curve_id(int curve_id, TPM2_ECC_CURVE *ret) {
+static const struct {
+        TPM2_ECC_CURVE tpm2_ecc_curve_id;
+        int openssl_ecc_curve_id;
+} TPM2_OPENSSL_ECC_CURVE_TABLE[] = {
```

Comment:
why uppercase? usually we don't uppercase our variable names

Reply:
ack, updated